test/files/run/stream-gc.scala

```diff
@@ -8,4 +8,5 @@ object Test extends App {
   Stream.tabulate(100)(_ => new Array[AnyRef](10000)).collectFirst { case x if false => x }
   Stream.tabulate(100)(_ => new Array[AnyRef](10000)).collectFirst { case x if false => x }
   Stream.tabulate(100)(_ => new Array[AnyRef](10000)).collectFirst { case x if false => x }
```

Comment:
late to the party, but why is this line repeated 3 times?

Reply:
I wondered, too. git says two of them were added in https://github.com/scala/scala/pull/9324/files#diff-b0ab17467318c0ab594f748c21870010636257c36577157fcdd7f01836f4b272. The same PR also updated the number of deprecation warnings, so it seems somewhat intentional, at least.

Reply:
@dwijnand ?

Reply:
That's funny - I have no recollection doing that.  Was I trying to stress the GC?  Did I do that before changing the java options and then committing?  I'd assume some sanity and say it's related to using JDK 15 in some way...  Maybe someone wants to try to retrace my steps.  I also don't mind if someone wants to remove the extras.